README.md

```diff
@@ -1,2 +1,73 @@
 # RoverSim
 Open Projects 2021
+
+## Abstract
+
+The project aims to introduce the Path-Planning phase to a wireless rover to make it capable to navigate autonomously in an environment with the help of path planning algorithms (Dijkstra and SLAM) parsed by a python script. It aims to reach the destination through the shortest path and shortest time. This was done using ROS navigation stack by performing a simulation of a rover model in Gazebo environment, by running ROS (Melodic Morenia) in Ubuntu 18.04. The robot used in this project is Turtlebot 3.
+
+![gazebo_sim](https://github.com/brahatesh/RoverSim/blob/main/Images%20and%20Videos/Gifs/gazebo_sim.gif "Simulation in Gazebo")
+
+![rviz_sim](https://github.com/brahatesh/RoverSim/blob/main/Images%20and%20Videos/Gifs/rviz_sim.gif "Simulation in RViz")
+
+
+## Motivation
+
+<p>Watching a robot move on its own without any human input is always fascinating. So, our team of 5 freshers decided to take on the challenge to learn how a robot maps the environment, find its position in the map of the environment and move from one position to another on its own.<br><br>The future will be fully automated with the help of robots and for that automated navigation will be extremely important. So making a small robot move automatically is the first step.</p>
+
+## Workflow
+
+![Workflow](https://github.com/brahatesh/RoverSim/blob/main/Images%20and%20Videos/Images/Workflow.jpg)
```

Comment:
Add a section named 'Software Aspect' and elaborate the Navigation & SLAM part briefly and hyperlink their README files here (https://github.com/brahatesh/RoverSim#readme-files-for-elaborative-explanation-of-working).